testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1249,12 +1220,12 @@ fn test_malformed_script() {
 #[ignore]
 fn test_key_manager_consensus_rotation() {
     // Create and launch a local validator swarm of 2 nodes.
-    let mut swarm = TestEnvironment::new(2);
-    swarm.launch_swarm(RoleType::Validator);
+    let mut env = TestEnvironment::new(2);
+    env.validator_swarm.launch();
```

Comment:
FYI: I know this test says ignore, but it now will completely not work :) @JoshLind Either you or me will have to come back and visit this test later.